tests/cupyx_tests/test_cupyx.py

```diff
@@ -10,9 +10,13 @@
 class TestErrState(unittest.TestCase):
 
     def test_errstate(self):
+        orig = cupyx.geterr()
         with cupyx.errstate(divide=self.divide):
             state = cupyx.geterr()
             assert state['divide'] == self.divide
+            for key in state:
+                if key != 'divide':
+                    assert state[key] == orig[key]
```

Comment:
How about simplifying as follows?
```suggestion
            assert state.pop('divide') == self.divide
            assert state == orig
```

Reply:
Looks good. I add `orig.pop('divide')` as `orig` also has `divide` key and we need to remove it to assert.